### ISSUE TITLE
EDScout crashes if a user's save games are relocated to another drive

### ISSUE THREAD

**Issue by joncage** [joncage/ed-scout#86]
```
E:\dstru\Downloads\EDScout-DiagnosticBuild\EDScout>EDScout.exe
Traceback (most recent call last):
  File "EDScoutWebUI\EDScout.py", line 217, in <module>
  File "EDScoutCore\EDScout.py", line 29, in __init__
  File "EDScoutCore\JournalInterface.py", line 168, in __init__
  File "EDScoutCore\JournalInterface.py", line 197, in _configure_watchers
Exception: Unable to start watching; Path does not exist: C:\Users\dstru\Saved Games/Frontier Developments/Elite Dangerous
[19600] Failed to execute script EDScout
```
The error is correct -- that path does not exist. My saved games are stored at E:\dstru\Saved Games using the standard Windows method of relocating user folders 
![image](https://user-images.githubusercontent.com/726660/93689009-1f903300-fa90-11ea-8daa-272817f57627.png)

This is the first application I've encountered with issues following the redirect and have been using this feature since Windows Vista.

_Originally posted by @Ketris in https://github.com/joncage/ed-scout/issues/77#issuecomment-695355062_

**Comment by joncage** [comment]
Looks like we can test if the directory exists and check for sum links as follows:

https://www.geeksforgeeks.org/python-os-path-islink-method/

**Comment by Ketris** [comment]
When using the OS-level folder move as screenshotted, no symlink or
shortcut is created. Literally nothing exists in c:\users\dstru\ but when
%userprofile% is opened as a folder in Explorer, the Saved Games folder
appears. I'm not sure how well Python can follow that behavior.

**Comment by joncage** [comment]
I'm sure there's a way!

**Comment by joncage** [comment]
> Literally nothing exists in c:\users\dstru\ but when %userprofile% is opened as a folder in Explorer, the Saved Games folder appears. I'm not sure how well Python can follow that behavior.

Can I just clarify, is it your whole user area that's been moved to `e:` or just the save games?

**Comment by Ketris** [comment]
Saved Games was moved individually. You cannot relocate the entire user
folder structure, only some of the folders.

**Comment by joncage** [comment]
Fixed in v1.5.3: https://github.com/joncage/ed-scout/releases/tag/v1.5.3
